tests/raster/test_geotiff.py

```diff
@@ -0,0 +1,307 @@
+import os
+import shutil
+import unittest
+import numpy as np
+import pandas as pd
+import xarray as xr
+from tempfile import mkdtemp
+
+from geospade.crs import SpatialRef
+from geospade.raster import Tile
+from geospade.raster import MosaicGeometry
+
+from veranda.raster.native.geotiff import GeoTiffFile
+from veranda.raster.mosaic.geotiff import GeoTiffReader, GeoTiffWriter
+
+
+class GeoTiffFileTest(unittest.TestCase):
+
+    def setUp(self):
+        """
+        Set up dummy mosaic set.
+        """
+        self.filepath = os.path.join(mkdtemp(), 'test.tif')
+
+    def test_read_write_1_band(self):
```

Comment:
you could also use pytest

```
@pytest.fixture
def filepath(tmp_path):
    return tmp_path / 'test.tif'

def test_read_write_1_band(filepath):
    ...
```

Reply:
What are pros and cons or is this just a kind of taste?

Reply:
Looked it up - I am now convinced to used `pytest`!